earth2studio/perturbation/hcbv.py

```diff
@@ -105,6 +106,9 @@ def create_generator(
             lead_time=input_coords["lead_time"],
             device="cpu",
         )
+        input_coords["time"] = warmup_times
+        input_data, data_coords = map_coords(input_data, data_coords, input_coords)
```

Comment:
Was graphcast a model where this would be needed? I'm trying to understand why the fetch data would not be sufficient. Thanks!